producer.py

```diff
@@ -1,11 +1,34 @@
-from json_asset import scrape
+from scraper import scrape, current_time
+from area_dict import area
 
-def producer():
-    raw_json = scrape('C03')
+current_time = current_time()
 
-    events = {}
+def producer(): #iterating each area in area_dict and returns a dictionary of all areas
+    all_areas = {}
+
+    for area_name in area.values():
+        area_code = area[area_name]
+        all_areas[area_code] = json_rearranger(area_code)
+    
+    return current_time, all_areas
```

Comment:
The var `current_time` is not in the function scope. Are you sure this will work as you expect?

Reply:
Also, where do you use this function?

Reply:
> Also, where do you use this function?

I am going to use it when storing data in JSON string into the SQL server. But I guess the function should be in `storer.py` or integrated into some another function in the file. will think.